### DISCUSSION TITLE
Issues Updating GitHub Authentication in Visual Studio

### DISCUSSION THREAD

**Discussion by Andrew-Downum-Vertex** [community/community#77124] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I'm currently having an issue attempting to connect GitHub Copilot to Visual Studio. 

I see these errors after installing the GitHub Copilot plugins:
 
![image](https://github.com/community/community/assets/135281553/fe8bd2cb-5ff6-435a-9967-2e7cdcdd67fa)

 
 
![image](https://github.com/community/community/assets/135281553/d2bcbea3-bf9a-468d-bc2a-f1d86682d0f9)

When I look at my credential page, I can see that I need to re-authenticate:
 
![image](https://github.com/community/community/assets/135281553/399539f2-4fad-4595-bfd3-3aa8dc1b4d8c)

 
but attempting to follow that flow results in a network timeout from localhost attempting to complete the OAuth flow:
 
![image](https://github.com/community/community/assets/135281553/d0b2dcac-8e0b-4756-b1cc-9369db3d6317)

url: https://localhost:59807/?browser_session_id=REDACTED&code=REDACTED&state=REDACTED

**Comment by asif-daffodil** [comment]
login to vs code with your GitHub account first. just try to use the commit and push option with with vscode first.  if everything is okay, then go tot your GitHub profile on the GitHub webpage and check that your GitHub copilot service is activated.

**Comment by johnmwright** [comment, marked as answer]
I was able to resolve this issue by going to `chrome://net-internals/#hsts` (I'm using the Chrome browser), then at the bottom, I entered `localhost` to the "delete" section and clicked the Add button.  After that, I was able to authenticate Copilot

You can find a few other things to try in this post, which is where I found the fix that worked for me: https://github.com/github/VisualStudio/issues/2521#issuecomment-733884398
